WA Intermission

Nelliel-ss14 opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->

Adds a new large ship service/medical ship named the Intermission

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Medical ship variety
## How to test
<!-- Describe the way it can be tested -->
Spawn into MD, buy Intermission ship
## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
<img width="1285" height="1103" alt="image" src="https://github.com/user-attachments/assets/ca9dd637-0780-4b6e-9a90-d07354a4156b" />
<img width="1335" height="977" alt="image" src="https://github.com/user-attachments/assets/52056402-b038-4ad8-a564-5b33f1c6a7ed" />
<img width="1249" height="1070" alt="image" src="https://github.com/user-attachments/assets/abd4df9c-94f2-4594-9fde-385eb9ddaee6" />
<img width="1436" height="1022" alt="image" src="https://github.com/user-attachments/assets/aa08b4e8-aee9-4ee2-bdab-49c02b919237" />

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [ X ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ X ] I have added media to this PR or it does not require an ingame showcase.
- [ X ] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
:cl:
add: The Intermission, new large service medical vessel fitted out to save lives and save appetites.

Nelliel-ss14 committed:
Respite, a small and cramped med ship

monolith8319 committed:
chore: automatically update REUSE headers

Nelliel-ss14 committed:
Merge remote-tracking branch 'upstream' into Nells-WA-Shipyards
Update branch to current

Nelliel-ss14 committed:
New Medical/Service Large Ship

Nelliel-ss14 committed:
Intermission Ship Playtest Fixes

Nelliel-ss14 committed:
Revert "Respite, a small and cramped med ship"

This reverts commit b6acd21d

monolith8319 committed:
chore: automatically update REUSE headers

Nelliel-ss14 commented:
Dont mind all the respite stuff, struggling with trying to remove it from my own branch as a 2nd day mapper, just the large ship files are in

OneZerooo0 reviewed (changes requested):
<img width="673" height="759" alt="image" src="https://github.com/user-attachments/assets/f8a3125b-5fef-4129-b11b-f414b03aae5b" />


Make this look better, could be done with some tasteful tile variety or partitions - In general, make the floors a bit more detailed and well done

Also, consider adding more EVA cabinets for more crew due to its size

OneZerooo0 commented:
Also, not sure where the WA comes from but SKR is starch's tag for ships he's designed, so maybe switch that one out or clear it with him first

Nelliel-ss14 committed:
More flooring tastes in canteen area, more EVA wall lockers, fixed ship tags

monolith8319 committed:
chore: automatically update REUSE headers

Nelliel-ss14 commented:
<img width="926" height="819" alt="image" src="https://github.com/user-attachments/assets/8385fc6b-e170-4576-96e8-c1838902ccb4" />

Canteen space has more floor variety, added 4 more EVA wall lockers to ship overall, and removed the SKR tag since it wasnt clear to me at the time that was Starch's tags and not just overall medical ship tagging

Nelliel-ss14 commented:
Also WA is a company that is currently in the works and not fully in yet, since its going to be a Black Market affiliated company, and from my knowledge Jester knows its going to be a BM company coming in, just not the name or full details of it

OneZerooo0 reviewed (approved):
eh I'm sure its fine
